tmt/steps/execute/__init__.py

```diff
@@ -186,14 +195,16 @@ class ExecutePlugin(tmt.steps.Plugin):
     """ Common parent of execute plugins """
 
     # List of all supported methods aggregated from all plugins
-    _supported_methods = []
+    _supported_methods: List[Method] = []
 
     # Common keys for all execute plugins
     _common_keys = ["exit-first"]
 
     # Internal executor is the default implementation
     how = 'tmt'
 
+    scripts: Optional[Tuple['Script', 'Script']] = None
```

Comment:
The number of scripts can vary, so this rather be the following?
```suggestion
    scripts: Tuple['Script', ...] = ()
```

Reply:
Updated, thanks!